<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,0,0">
    <item>
     <widget class="QTreeView" name="treeFileView"/>
    </item>
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="MainPlainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="create"/>
    <addaction name="open"/>
    <addaction name="openReadOnly"/>
    <addaction name="save"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="options">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="control">
     <property name="title">
      <string>Управление</string>
     </property>
     <addaction name="buttons"/>
    </widget>
    <widget class="QMenu" name="language">
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="russian"/>
     <addaction name="english"/>
    </widget>
    <widget class="QMenu" name="theme">
     <property name="title">
      <string>Тема</string>
     </property>
     <addaction name="light"/>
     <addaction name="dark"/>
    </widget>
    <addaction name="control"/>
    <addaction name="language"/>
    <addaction name="theme"/>
   </widget>
   <widget class="QMenu" name="faq">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutProgram"/>
   </widget>
   <widget class="QMenu" name="view">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="treeViewShow"/>
   </widget>
   <addaction name="file"/>
   <addaction name="view"/>
   <addaction name="options"/>
   <addaction name="faq"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="openReadOnly">
   <property name="text">
    <string>Открыть для чтения</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="buttons">
   <property name="text">
    <string>Клавиши</string>
   </property>
  </action>
  <action name="russian">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="english">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="light">
   <property name="text">
    <string>Светлая</string>
   </property>
  </action>
  <action name="dark">
   <property name="text">
    <string>Тёмная</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="aboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="treeViewShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Проводник</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
